src/routes/webhooks/paddle.ts

```diff
@@ -93,7 +97,7 @@ const getUserId = async ({
   subscriptionId,
   userId,
 }: {
-  subscriptionId?: false | string | null;
+  subscriptionId?: false | string | null | unknown;
```

Comment:
Why this? When you add unknown you are effectively saying I don't know the type so the other types are not really needed anymore.

Do we know this type?

Reply:
From the last addition it just became unknown type in their types.
For the others we do know and didn't need the unknown 😢

Reply:
My bad was wrong field in this type